runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -185,6 +161,38 @@ public void addNode(Layout currentLayout,
         reconfigureSequencerServers(currentLayout, newLayout, false);
     }
 
+    /**
+     * Heals an existing node in the layout.
+     *
+     * @param currentLayout Current layout.
+     * @param endpoint      New endpoint to be added.
+     * @throws QuorumUnreachableException if seal or consensus cannot be achieved.
+     * @throws OutrankedException         if consensus outranked.
+     */
+    public void healNode(Layout currentLayout,
+                         String endpoint)
+            throws QuorumUnreachableException, OutrankedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.runtime.exceptions.QuorumUnreachableException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)